Makefile

```diff
@@ -407,6 +390,18 @@ internal-clean: $$(KIND)
 internal-distclean: $$(KIND)
 	@rm -rf ./_build/$(KIND)
 
+internal-ct: $$(KIND)
+internal-ct: $$(SYSCONFIG)
+internal-ct: $$(AETERNITY_TESTCONFIG_DB_BACKEND)
```

Comment:
Is this a prerequisite on a target named e.g. leveled?

Reply:
No. What would that target do?

Reply:
I spoke with @tolbrino , this line seems unnecessary.